apps/web/components/ServiceInformation/ServiceInformation.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import * as styles from './ServiceInformation.treat'
+import {
+  Box,
+  GridColumn,
+  GridRow,
+  Inline,
+  Tag,
+  Text,
+} from '@island.is/island-ui/core'
+import { ApiService } from '@island.is/api/schema'
+import { GetNamespaceQuery } from '@island.is/web/graphql/schema'
+import { useNamespace } from '../../hooks'
+
+export interface ServiceInformationProps {
+  service: ApiService
+  strings: GetNamespaceQuery['getNamespace']
+}
+
+export const ServiceInformation = ({
+  service,
+  strings,
+}: ServiceInformationProps) => {
+  const n = useNamespace(strings)
+
+  const capitalize = (s: string) => {
+    if (typeof s !== 'string') return ''
+
+    return s.charAt(0).toUpperCase() + s.slice(1).toLowerCase()
+  }
+
+  const horizontalLine = () => {
+    return <Box className={styles.underLine} />
+  }
+
+  const showDesktopTags = () => {
+    return (
+      <Box>
+        <GridRow>
+          <GridColumn span={['12/12', '12/12', '6/12']} paddingTop="gutter">
+            <Box className={styles.underLine} width="full">
+              <Text variant="h3">{n('data')}</Text>
+            </Box>
+            <Inline space={1}>
+              {service.data?.map((item) => (
+                <Tag variant="white" outlined>
+                  {n(`data${capitalize(item)}`)}
+                </Tag>
+              ))}
+            </Inline>
+          </GridColumn>
+          <GridColumn span={['6/12', '6/12', '3/12']} paddingTop="gutter">
+            <Box className={styles.underLine} width="full">
+              <Text variant="h3">{n('type')}</Text>
```

Comment:
I'm probably not going to comment on more `Text` component using header that need the `as` prop but this should apply to all `Text` headers
```suggestion
              <Text variant="h3" as="h3">{n('type')}</Text>
```